src/vis/density/PieChart.tsx

```diff
@@ -0,0 +1,34 @@
+import * as d3 from 'd3v7';
+import { uniqueId } from 'lodash';
+import * as React from 'react';
+import { useMemo } from 'react';
+
+export interface PieChartProps {
+  data: number[];
+  dataCategories: string[];
+  radius: number;
+  transform: string;
+  colorScale: d3.ScaleOrdinal<string, string, never>;
+}
+
+export function PieChart({ data, dataCategories, radius, transform, colorScale }: PieChartProps) {
+  const pie = useMemo(() => {
+    return d3.pie();
+  }, []);
+
+  const arc = useMemo(() => {
+    return d3.arc().innerRadius(0).outerRadius(radius);
+  }, [radius]);
+
+  const id = React.useMemo(() => uniqueId('PieNum'), []);
+
+  return (
+    <g style={{ transform }}>
+      {pie(data).map((slice, i) => {
+        // TODO: Why are indexes bad in the key? how else to do this? Also, I think the typings for arc are wrong, which is why im typing slice to any
+        // eslint-disable-next-line react/no-array-index-key
+        return <path key={`${id}, ${i}`} d={arc(slice as any)} style={{ fill: colorScale ? colorScale(dataCategories[i]) : 'cornflowerblue' }} />;
```

Comment:
Maybe you can apply a similar solution as proposed in [this article](https://medium.com/stationfive/how-to-create-a-pie-chart-with-d3-js-and-react-hooks-part-1-81bcd7f39b32) (jump to _The simplest way using SVG elements_)?

Additionally, locally my ESLint was not complaining when I removed the eslint-disable line.